cypress/support/commands.ts

```diff
@@ -297,6 +297,10 @@ export const upsertSecret = (
   })
 }
 
+export const clickNavBarItem = (testID: string): Cypress.Chainable => {
+  return cy.getByTestID(testID).click({force: true})
+}
```

Comment:
this is ok for now - but I was hoping this function would actually do the work to expand the nav bar menu and then select the menu item

Reply:
Ah! I see what you mean. I think me and David thought the same thing that we just want to wrap it up into a function.

Reply:
The reason I think it would be a good idea to actually expand the menu and then click the visible nav item is because that's how the users will do it real life. But it's not a big deal - now that you've put it in a function you could always make that improvement later! Now it only has to be changed in one spot 😄

Reply:
Maybe put a "//TODO" comment in the function for now or something